en/configure-a-tidb-cluster.md

```diff
@@ -127,6 +127,22 @@ If you want to enable TiCDC in the cluster, you can add TiCDC spec to the `TiDBC
       replicas: 3
```

Comment:
`deploy-on-general-kubernetes.md` has been split up, and some of the contents were moved into `configure-a-tidb-cluster.md`. So the changes of `zh/deploy-on-general-kubernetes.md` in the origin PR are added to this file.